operate/client/src/App/ProcessInstance/TopPanel/index.tsx

```diff
@@ -243,7 +245,7 @@ const TopPanel: React.FC = observer(() => {
             >
               {stateOverlays.map((overlay) => {
                 const payload = overlay.payload as {
-                  flowNodeState: FlowNodeState;
+                  flowNodeState: FlowNodeState | 'completedEndEvents';
```

Comment:
Is there a specific reason to add `completedEndEvents` as an "external addition" instead of adding it as an option in the `FlowNodeState` type definiton?

Reply:
It's a bit tricky. The type `FlowNodeState` is used in multiple places and reflects the backend API type. However the distinction between `completed` and `completedEndEvents` is made in FE code only. That's why I decided to keep it local in `processInstanceDetailsStatistics.ts`.